package.json

```diff
@@ -59,12 +59,15 @@
     ]
   },
   "dependencies": {
+    "@types/bunyan": "^1.8.0",
```

Comment:
I agree. We shouldn't be adding typescript to the dependencies since we are using flow already.  They are pretty much the same so your example shouldn't have to change at all.

If you want to use typescript in your own codebase it's fine but here it's all flow. You could do a PR with a typescript definition if you wanted.